src/mealDb.js

```diff
@@ -0,0 +1,181 @@
+            /* eslint-disable-next-line */
+            .map(([key, value]) => value);
+          for (let i = 1; i < ingredientsArray.length; i += 1) {
+            ingredientsUl.innerHTML += `<li> ${ingredientsArray[i]} - ${measuresArray[i]} </li> `;
+          }
+        };
+        ingredients();
+        const exit = document.querySelector('.fa-times');
+        exit.addEventListener('click', () => {
+          header.classList.remove('hidden');
+          main.classList.remove('hidden');
+          footer.classList.remove('hidden');
+          modal.classList.add('hidden');
+        });
+      }
+      sendComment();
+    });
+  };
+
+  for (let i = 0; i < btn.length; i += 1) {
+    btn[i].addEventListener('click', () => {
+      window.scrollTo(0, 0);
+      const id = btn[i].getAttribute('data');
+      dataModal(id);
+      header.classList.add('hidden');
+      main.classList.add('hidden');
+      footer.classList.add('hidden');
+      modal.classList.remove('hidden');
+    });
+  }
+};
+
+// Displaying meal
+const fetchMeal = async () => {
+  const meallist = document.querySelector('.meal-list');
+  const getCategoryUrl = async (category) => {
+    meallist.innerHTML = '';
+    const printMeals = (meals) => {
+      meals.forEach((element) => {
+        const newmeal = document.createElement('li');
+        newmeal.innerHTML = ` 
+        <div class="meal">
+        <div class="meal-header">
+          <img src="${element.strMealThumb}" loading="lazy" width="500" height="280" alt="${element.strMeal}">
+        </div>
+        <div class="meal-body d-flex justify-between">
+          <h4>${element.strMeal}</h4>
+          <button class="fav-btn" ><i class="fas fa-heart" data="${element.idMeal}"></i></button>
+        </div>
+        <div>
+          <div class = "likes"><span class= "likes-qty" data="${element.idMeal}">0</span> likes </div>
+          <button class= "comments" data="${element.idMeal}">Comments</button>
+        </div>
+      </div>`;
+
+        meallist.appendChild(newmeal);
+      });
+    };
+
+    const mealapiurl = `https://www.themealdb.com/api/json/v1/1/filter.php?c=${category}`;
+    const meals = await fetch(mealapiurl)
+      .then((res) => res.json())
+      .then((data) => data.meals);
+    printMeals(meals);
+    popup();
+  };
+
+  const foodCount = document.getElementsByClassName('foodCount');
+  const printCount = async (category) => {
+    const mealapiurl = `https://www.themealdb.com/api/json/v1/1/filter.php?c=${category}`;
+    const meals = await fetch(mealapiurl)
+      .then((res) => res.json())
+      .then((data) => data.meals);
+
+    for (let i = 0; i < foodCount.length; i += 1) {
+      foodCount[i].innerHTML = '';
+      if (foodCount[i].classList.contains(category)) {
+        foodCount[i].innerHTML = `(${meals.length})`;
+      }
+    }
+  };
+
+  const navlinks = document.querySelectorAll('nav li');
+  const resetLinks = () => {
+    for (let i = 0; i < navlinks.length; i += 1) {
```

Comment:
- [Optional] Nice work here, for better use of ES6 Methods you can use a `ForEach` or `map` instead of a for loop.

Reply:
Thank you for guidance.